tests/data/standard_scenarios_registration.json

```diff
@@ -0,0 +1,33 @@
+{
+  "create_registry": true,
+  "registry_path": "updated-standard-scenarios-registry",
+  "projects": [
+    {
+      "project_id": "dsgrid_conus_2022",
+      "config_file": "../dsgrid-project-StandardScenarios/dsgrid_project/project.toml",
+      "register_project": true,
+      "datasets": [
+        {
+          "dataset_id": "conus_2022_reference_comstock",
+          "dataset_path": "../dsgrid-data/conus_2022_reference_comstock",
+          "config_file": "../dsgrid-project-StandardScenarios/dsgrid_project/datasets/modeled/comstock/dataset.toml",
+          "dimension_mapping_file": "../dsgrid-project-StandardScenarios/dsgrid_project/datasets/modeled/comstock/dimension_mappings.toml",
+          "register_dataset": true,
+          "submit_to_project": true,
+          "fix_dimension_uuids": false,
```

Comment:
what does the fix refer to? Replacing the uuids between config files automatically after registration?

Reply:
Suppose a dataset uses a dimension created by the project. The dataset contributor has to put the dimension_id (with UUID) in the file as a dimension reference. If the project gets re-created, that UUID will be wrong. We have code that will update the UUIDs.